<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiMainWindow</class>
 <widget class="QMainWindow" name="GuiMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="widgetEmpty">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelLogo">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="rsrc.qrc">:/pics/logo.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="MACHWidget" name="widgetMACHO"/>
      <widget class="MACHOFATWidget" name="widgetMACHOFAT"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1320</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MACHWidget</class>
   <extends>QWidget</extends>
   <header location="global">machwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MACHOFATWidget</class>
   <extends>QWidget</extends>
   <header>machofatwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rsrc.qrc"/>
 </resources>
 <connections/>
</ui>
